codecov.yml

```diff
@@ -1,16 +0,0 @@
-# https://docs.codecov.io/docs/codecov-yaml
```

Comment:
Why was this removed?

Reply:
Its dead code.  We haven't been able to get code coverage results since we switched to AzDO.  I can add it back.

Reply:
It's unrelated to the change, I was kinda expecting that we would bring Code Cov back, I could have sworn you had a PR that was attempting to do this.

Reply:
I tried... No one has figured out how to do it for AzDO yet.

I'll bring this back anyways.  You are right that its unrelated